fleet/fleet.go

```diff
@@ -0,0 +1,87 @@
+// Package fleet implements a fleet client providing basic operations against a
+// fleet endpoint through fleet's HTTP API. Higher level scheduling and
+// management should be built on top of that.
+package fleet
+
+import (
+	"net"
+	"net/http"
+
+	"github.com/coreos/fleet/client"
+	"github.com/coreos/fleet/etcd"
+	"github.com/coreos/fleet/machine"
+	"github.com/coreos/fleet/registry"
+	"github.com/coreos/fleet/schema"
+	"github.com/coreos/fleet/unit"
+)
+
+// TODO
+type Config struct{}
+
+// TODO
+func DefaultConfig() Config {
+	newConfig := Config{}
+
+	return newConfig
+}
+
+type Status struct {
+	// SystemdStatus represents the status within systemd.
+	SystemdStatus containerstates.State
+
+	// FleetStatus represents the status within fleet.
+	FleetStatus containerstates.State
```

Comment:
Based on this documentation I neither see that there are the three specific ones you are talking about nor there are 5 in total. I am still confused. Anyway, `FleetDesired` and `FleetCurrent` makes sense. What is ActiveState? Is it SystemdCurrent?